src/items/static-items.md

```diff
@@ -6,18 +6,17 @@
 >              `=` [_Expression_] `;`
 
 A *static item* is similar to a [constant], except that it represents a precise
```

Comment:
This paragraph is just a list of properties, which makes me feel like it does not flow very well. I'm not sure the best way to address that; perhaps splitting the things about lifetimes into a separate paragraph would help, or adding more connective prose?

Reply:
It definitely doesn't flow well. It reads almost like a list but without the bullet points. Should probably be addressed, but not as part of this issue.